mm2src/coins/eth.rs

```diff
@@ -2257,7 +2257,7 @@ impl MarketCoinOps for EthCoin {
         }
     }
 
-    fn wait_for_confirmations(&self, input: ConfirmPaymentInput) -> Box<dyn Future<Item = (), Error = String> + Send> {
+    fn wait_for_confirmations(&self, input: ConfirmPaymentInput) -> Box<dyn Future<Item = u64, Error = String> + Send> {
```

Comment:
why not `u8` or `u16`?

Reply:
too keep consistence with the field `pub confirmations: u64,`